packages/@divvi/mobile/src/redux/store.ts

```diff
@@ -121,6 +122,8 @@ export const setupStore = (initialState?: ReducersRootState, config = persistCon
         immutableCheck: false,
         serializableCheck: false,
       }).concat(...middlewares),
+    devTools: false,
+    enhancers: (getDefaultEnhancers) => getDefaultEnhancers().concat(devToolsEnhancer()),
```

Comment:
Does this only activate in `__DEV__`?

Reply:
Answering myself, yes it's fine, it's a noop when `process.env.NODE_ENV === "production"`

https://github.com/matt-oakes/redux-devtools-expo-dev-plugin/blob/927766e5206901178c1d335daa9bd8b0ea806001/src/index.ts#L10-L13